### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by Tinacyt** [microsoft/testbase#5] [Announcements]
We’re using Discussions as a place to connect with other members of our Test Base community. 😊

We hope that you:
🗨️Provide feedback on Test Base experience (Azure portal, sample packages, SDK samples...)
❓Ask questions you’re wondering about
💡Share ideas
🤝Engage with other community members
🎶Welcome others and are open-minded

Remember that this is a community we build together.🎉

**Comment by swahelamulla** [comment]
I'm trying to upload my package to azure test base using azure rest API, Every time package get's uploaded successfully to test base service but  package get  failed (every time verification status is verification failed ) showing some blob related error message.("The specified blob does not exist.") I have taken blob path from file upload URL of a Test Base Account.(I have stored my package inside storage account with a container have public access)

When I run rest API the Response Code: 201

Error in JSON File after failure in verification.

"packageStatus": "Error",
"validationResults": [
            {
                "validationName": "Sanity Checks Validator",
                "isValid": false,
                "errors": [
                    "The specified blob does not exist."
                ]
            }
        ],
 "validationRunStatus": "Failed"

**Reply by Tinacyt** [reply]
Hi there, 

Thank you for using Test Base and reporting this issue😊.

We'd like to better support you with more detailed information! It shall be great if you can create a support ticket on Azure Portal by following the guidance [https://docs.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request](url). 

Warm Regards,
Tina

**Reply by swahelamulla** [reply]
Thank you!
